.github/workflows/deps_lint.yml

```
@@ -87,6 +87,31 @@ jobs:
         .ci/openapi_diff.sh
         .ci/index_openapi_diff.sh
 
+  docker-image:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build the Docker image
+      run: docker-compose build
+
+    - name: Test server
+      run: |
+        docker-compose up optimade &
+        .github/workflows/wait_for_it.sh localhost:3213 -t 120
+        sleep 15
+        curl http://localhost:3213/optimade/info > info.json
+        grep -F "http://example.com" info.json || exit 1
```

```suggestion
        grep -F "http://example.com" info.json || exit 1
        grep -F '"data":{' info.json || exit 1
```

Just a suggestion that distinguishes between a malformed URL and a working one. I'm sure there's a more relevant test string to validate against.

---

He, I did this as well in my own implementation, and then changed it again to something even more dastardly, see [here](https://github.com/aiidateam/aiida-optimade/blob/82d0406be21811fb50b8c4838ddade8da728082d/.github/workflows/ci.yml#L175) :)

---

Going for your other suggestions :+1: